packages/playground/tests/utils/isValidSmtp.test.ts

```diff
@@ -0,0 +1,24 @@
+import { describe, expect, it } from "vitest";
+
+import { isValidSmtp } from "../../src/utils/validators";
+
+describe("isValidSmtp", () => {
+  it("returns nothing for valid username and email", () => {
```

Comment:
I truly don't understand can you point out the `username` and the `email` because there's only just one input name `input` which looks like an email.

Reply:
done in the current validation rules.